src/Investments/index.jsx

```diff
@@ -1,17 +1,40 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useEffect } from 'react';
+import { View } from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchUserInvestmentShares, fetchStockPrice } from '../redux/investment/actionCreator';
 import commonStyles from '../common/styles';
 import SearchButton from '../common/HeaderSearchButton';
+import PortfolioValue from './PortfolioValue';
 
 const navigationOptions = {
+  title: 'Investment',
   headerRight: () => <SearchButton />,
 };
 
-const Investment = () => (
-  <View style={commonStyles.container}>
-    <Text>This is the investment screen</Text>
-  </View>
-);
+const Investment = () => {
+  const dispatch = useDispatch();
+  const {
+    isLoading,
+    userShares,
+  } = useSelector((state) => state.investments);
+
+  useEffect(() => {
+    dispatch(fetchUserInvestmentShares(1));
```

Comment:
What is this magic number? The userId? 
In this case it should come from the redux store or as a navigation param, lets discuss it with the team. I am okay with redux store but in this case we have to store it during the login

Reply:
Also if the team decides to postpone this, then let's go with hardcoded userID and create a jira story to make the investment screen dynamic